packages/components/src/TokenTutorial/TokenTutorialLanding.tsx

```diff
@@ -70,29 +73,76 @@ export class TokenTutorialLanding extends React.Component<TokenTutorialLandingPr
           </TakeQuizBtn>
         </TutorialSkipSection>
 
-        {TutorialContent.map((topic, idx) => (
-          <TutorialTopic key={idx}>
-            <LaunchTopic onClick={() => this.openTutorial(idx)}>
-              <div>
-                {topic.icon}
-                <h3>{topic.name}</h3>
-                <p>{topic.description}</p>
-              </div>
-              <DisclosureArrowIcon />
-            </LaunchTopic>
-            <TopicProgress>
-              <TutorialProgressText questions={topic.questions.length} />
-              <TutorialLandingProgressBars>
-                {topic.questions.map((x, i) => <TutorialLandingProgressBar key={i} />)}
-                <b>0/{topic.questions.length}</b>
-              </TutorialLandingProgressBars>
-            </TopicProgress>
-          </TutorialTopic>
-        ))}
+        {TutorialContent.map((topic, idx) => {
+          const { isComplete, lastSlideIdx } = this.getTopicStatus(this.props.quizPayload, topic);
+
+          // TODO(jorgelo): What do we do when isComplete is true (this means that this topic has been completed)
+          // TODO(jorgelo): lastSlideIdx is the last slide that was completed correctly. Should we jump the user to that last slide?
```

Comment:
@jorgelo I think if you pass in `quizSlide` from the landing page it should work. You would still see the tutorial but then it should take you to the question you left off on.